web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -62,4 +63,55 @@ describe("<SidebarCardsList />", () => {
     fireEvent.click(screen.getByTestId("hidden-opportunity-header"));
     expect(mockAnalytics.event.for_you_card_unhide_button.click.unhide_cards).toHaveBeenCalledTimes(1);
   });
+
+  describe("empty state messaging", () => {
```

Comment:
[pebble] We had talked about how some of the names in `SidebarCardsList` may have been slightly confusing, and because the meanings of those names would be changing with this work it would be a good opportunity to rename. One of which was the function `showEmptyState`. I see that function is still in the `SidebarCardsList` component. By calling this describe "empty state messaging" it almost seems like we're testing different functionality.

I'm not saying this name has to change, but let's make sure that this describe title is reflective of the named behavior we want to test.

I think when we originally named this section we were trying to use "empty state messaging" as an all-encompassing way to cover both of the For You messages. Let's consider that in whatever name we use for this describe or as a guide for renaming the functionality in the component.

Reply:
[boulder] I would love to see the other message behavior currently covered by the function `showEmptyState` tested as well. It's not specifically covered by this ticket, but we are making changes to this component and the tests shouldn't take much effort. Additionally, we had questions about the intended behavior of that message in relation to this new work. Those tests will serve as documentation for us and help prevent this confusion in the future.